R/eval.R

```diff
@@ -32,6 +32,9 @@ rust_eval <- function(code, env = parent.frame(), ...) {
   # make sure code is given as a single character string
   code <- glue::glue_collapse(code, sep = "\n")
 
+  # define to make R code check happy; is not used
+  rextendr_rust_eval_fun <- function() "decoy function; should never be called."
```

Comment:
Perhaps this message should be put into `stop()`?
If it gets called (somehow), the output may be swallowed, but an explicit error will indicate something is not right.

Reply:
I'll make that change.